src/pybullet_helpers/joint.py

```diff
@@ -0,0 +1,103 @@
+"""Pybullet helper class for joint utilities."""
+from typing import List, NamedTuple, Sequence, Tuple
+
+import numpy as np
+import pybullet as p
+
+from predicators.src.pybullet_helpers.geometry import Pose3D, Quaternion
+
+
+class JointInfo(NamedTuple):
+    """Joint Information from Pybullet."""
+    jointIndex: int
```

Comment:
is the reason for using camelCase here to match the names of pybullet attributes? if so, that's fine

Reply:
but also let's point out the closest pybullet function(s) that these variable names are meant to match, for reference (`p.getJointInfo` probably)

Reply:
Yes to match pybullet, will update docstring